src/button.js

```diff
@@ -84,12 +84,11 @@ class Button extends Sprite.class {
     button.style = srOnlyStyle;
     button.textContent = this.text;
 
-    /**
-     * If the button is disabled.
-     * @memberof Button
-     * @property {Boolean} disabled
-     */
-    this.disabled = this._dn.disabled = disabled;
+    if (disabled) {
+      this.disable();
+    } else {
+      this.enable();
+    }
```

Comment:
Since the button is enabled by default, we actually don't need to enable it. Was the test that is failing the one you added to check default props? If so you could remove that one as `undefined` is equivalent to `false` for the purposes of being disabled

Reply:
Ah I see, apologies. I've made these changes now.